### ISSUE TITLE
[ONNX] Support ConvTranspose + auto_pad

### ISSUE THREAD

**Issue by stakemura** [apache/incubator-tvm#3608]
TVM Relay (latest master branch) couldn't compile ONNX including [ConvTranspose + auto_pad](https://github.com/onnx/onnx/blob/master/docs/Operators.md#ConvTranspose).

Here is the code and the attached 
[ZIP-compressed ONNX file updated on Jul 25](https://github.com/dmlc/tvm/files/3429710/upconvert.onnx.zip)
 to reproduce the error.

```
import tvm
import tvm.relay as relay
import onnx

onnx_model = onnx.load('upconvert.onnx')
 
shape_dict = {'input__0': (1, 3, 360, 640)}
mod, params = relay.frontend.from_onnx(onnx_model, shape_dict)
```

This issue is critical for us.
Would you like to support this attribute?

**Comment by tqchen** [comment]
It would be great if you can look a bit into what is going on, and the community can provide help on possibly actions

**Comment by vinx13** [comment]
@stakemura this means we need to infer padding in case `auto_pad` is set in https://github.com/dmlc/tvm/blob/e7fb2d4d05858ad64f27cc06b944d5abcdaaef3e/python/tvm/relay/frontend/onnx.py#L209

**Comment by stakemura** [comment]
@vinx13 Thank you for the supplementary explanation!
@abuccts I deeply appreciate your help. I would like to try your patch.

**Comment by abuccts** [comment]
Hi @stakemura, could you help to verify the patch? @vinx13 could you please review the PR, thx!

**Comment by stakemura** [comment]
@abuccts I'm sorry for the late reply. I confirmed #3645 patch can load the attached onnx model successfully.

**Comment by cchung100m** [comment]
Hi @zhiics 

This PR should be closed by #4563, too. I would appreciate that if you can help to manage it, thanks :)

**Comment by zhiics** [comment]
@cchung100m Thanks for resolving the issue.
